app/assets/stylesheets/darkswarm/cookies_banner.css.scss

```diff
@@ -0,0 +1,30 @@
+@import 'branding';
+
+.cookies-banner {
+  background: $dark-grey;
+  bottom: 0;
+  position: fixed;
+  top: 20vh !important;
```

Comment:
What browser versions do we aim to support? I happened to check this recently: https://caniuse.com/#feat=viewport-units

Reply:
I don't think we have a set of supported browsers. It would be good to create that list though. I'll post something in Slack.